design/www/app/common/tost.service.js

```diff
@@ -0,0 +1,20 @@
+ (function() {
+    'use strict';
+    angular.module('starter')
+            .factory('tostService', tostService);
+
+    function tostService() {
+        return {
+            notify: function(message, position) {
+                window.plugins.toast.showWithOptions(
```

Comment:
put a check 
if(window.plugins && window.plugins.toast){
 ///then execute ur code
}else{
  // this means on browser simply alert the message
}